socketcan_interface/package.xml

```diff
@@ -13,16 +13,17 @@
   <url type="repository">https://github.com/ros-industrial/ros_canopen</url>
   <url type="bugtracker">https://github.com/ros-industrial/ros_canopen/issues</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
 
+  <depend>pluginlib</depend> 
   <depend>boost</depend>
-  <depend>class_loader</depend>
-  <depend>libconsole-bridge-dev</depend>
```

Comment:
why did you drop  `libconsole-bridge-dev`?

Reply:
I'm not sure why I dropped it. Re-added.